cfgov/jobmanager/blocks.py

```diff
@@ -1,80 +1,72 @@
-from django.template.loader import render_to_string
-from django.utils import timezone
-
 from wagtail.core import blocks
 
-from v1.atomic_elements import organisms
+from jobmanager.models import JobListingPage
 from v1.util.util import extended_strftime
 
 
-class OpenJobListingsMixin(object):
-    def filter_queryset(self, qs, value):
-        qs = super(OpenJobListingsMixin, self).filter_queryset(qs, value)
-
-        # Hide any jobs that have not been published.
-        qs = qs.filter(live=True)
-
-        if value.get('hide_closed'):
-            today = timezone.now().date()
-            qs = qs.filter(open_date__lte=today, close_date__gte=today)
-
-        return qs
+class JobListingsMixin:
+    def get_context(self, value, parent_context=None):
+        context = super().get_context(value, parent_context=parent_context)
+        context.update({
+            'jobs': self.get_job_listings(),
+            'no_jobs_message': (
+                'There are no current openings at this time.'
```

Comment:
This is nitpicky, but "at this time" is redundant after saying "There are no current openings."

If this is cleared content that can't be changed without approval, ignore me.